test/tests/treeview_treeview-1a.js

```diff
@@ -80,6 +80,60 @@ const isClosedFolderTreeitem = async function (el) {
   return (await el.getAttribute('aria-expanded')) === 'false';
 };
 
+ariaTest(
+  'aria-selected attribute on treeitem initial value',
+  exampleFile,
+  'treeitem-aria-selected',
+  async (t) => {
+    const treeitems = await t.context.queryElements(t, ex.treeitemSelector);
+
+    for (let treeitem of treeitems) {
+      t.is(
+        await treeitem.getAttribute('aria-selected'),
+        'false',
+        'treeitem should initially have aria-selected="false"'
+      );
+    }
+  }
+);
+
+ariaTest(
+  'aria-selected attribute on treeitem on down arrow and enter',
+  exampleFile,
+  'treeitem-aria-selected',
+  async (t) => {
+    const items = await t.context.queryElements(t, ex.treeitemSelector);
+
+    for (let i = 0; i < items.length - 1; i++) {
+      await items[i].sendKeys(Key.ARROW_DOWN);
+      const nextItem = await t.context.queryElement(
+        t,
+        ex.treeitemSelector + '[tabindex="0"]'
+      );
+      const focusMoved = items[i] !== nextItem;
```

Comment:
As an aside this will always be true. As I understand it, `nextItem` may represent the same element in the dom but webdriver's js api doesn't guarentee that it will be the same object in this context. To test that they are the same I think you'd use `WebElement.equals(items[i], nextItem)` or `(await items[i].getId()) !== (await nextItem.getId())`.